api/src/pcapi/core/offers/models.py

```diff
@@ -176,6 +176,7 @@ class Stock(PcObject, Base, Model, ProvidableMixin, SoftDeletableMixin):
     priceCategory: sa_orm.Mapped["PriceCategory | None"] = sa.orm.relationship("PriceCategory", back_populates="stocks")
     quantity = sa.Column(sa.Integer, nullable=True)
     rawProviderQuantity = sa.Column(sa.Integer, nullable=True)
+    features = sa.Column(postgresql.ARRAY(sa.Text), nullable=True)
```

Comment:
Il serait pratique d'avoir une liste vide comme valeur par défaut et de rendre le champ NOT NULLable. Cela évitera d'avoir à écrire des trucs comme : 

```python
if stock.features and "x" in stock.features:
    # bla bla bla

# ou bien :
if "x" in (stock.features or []):
    # bla bla bla    pass
```

Avec une valeur par défaut, on est certain d'avoir un seul type de données : une liste.